libs/transactions-journey/e2e-tests/locators/transaction-details.locators.ts

```diff
@@ -0,0 +1,5 @@
+export const TRANSACTION_DETAILS_LOCATORS = {
```

Comment:
Was there any reason for having the locators files separate from the page objects? I am not clear on the purpose of this.

Reply:
i my opinion, it can be single file for all transactions. It gets easier to update in case of component changes when all locators are in single file, instead of it being present in the tests itself. wdyt?

Reply:
Sounds good to me, so long as it make sense for everyone else :)